pyproject.toml

```
@@ -0,0 +1,129 @@
+# For more information about configuring project metadata for the
+# setuptools build backend, please see
+# https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools"]
+
+[project]
+authors = [
+    { name = "Cybersecurity and Infrastructure Security Agency", email = "github@cisa.dhs.gov" }
+]
+# See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers = [
+    # How mature is this project? Common values are
+    #   3 - Alpha
+    #   4 - Beta
+    #   5 - Production/Stable
+    "Development Status :: 3 - Alpha",
+    "Environment :: Console",
+    # Indicate who your project is intended for
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    # Specify the Python versions you support here. In particular, ensure
+    # that you indicate whether you support Python 2, Python 3 or both.
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
+    "Programming Language :: Python :: Implementation :: CPython",
+]
+dependencies = [
+    "actions-toolkit",
+    "chevron",
+    "docopt",
+    "PyGithub",
+    "PyYAML",
+    "schema",
+]
+description = "GitHub Action to automatically generate PR requests from upstream repositories regardless of the fork network."
```

Improve description grammar:
```suggestion
description = "GitHub Action to automatically generate pull requests from upstream repositories regardless of the fork network."
```